queuebacca-core/src/main/java/io/axonif/queuebacca/ScopedMessageConsumer.java

```diff
@@ -67,11 +67,11 @@
 
 	/**
 	 * Creates a new instance of a {@link ScopedMessageConsumer} with a {@link MessageConsumer} it will delegate to,
-	 * and one or more {@link MessageScope PreProcessors}.
+	 * and one or more {@link MessageScope Processors}.
```

Comment:
These should be MessageScopes. Originally they were called PreProcessors to create a dichotomy with PostProcessors. I likely just refactor-renamed them to MessageScope and forgot to scan over the comments.